tests/e2e-test.spec.js

```diff
@@ -59,20 +60,30 @@ const waitForCondition = async (condition, timeout = 20000, interval = 0.1) => {
 
 describe('Main workflow Test Suite', () => {
   let client;
+  let sshServer;
+  let tunnel;
 
   before(async () => {
     console.log('Importing docs');
     await importAllDocs();
+    console.log('Creating SSH tunnel');
+    tunnel = await setupTunnel();
```

Comment:
🤩

Reply:
Thanks for the review!

I'll resolve this compliment (thanks!) and then leave the PR open for another day or so in case @njuguna-n or @witash have any comments.